public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,111 @@
+### `dvc.api.get_url(path, repo=None, rev=None, remote=None)`
+
+Returns the URL to the storage location of a data file or directory tracked by
+DVC.
+
+```py
+import dvc.api
+
+resource_url = dvc.api.get_url(
+    'get-started/data.xml',
+    repo='https://github.com/iterative/dataset-registry')
+
+# resource_url =
+# https://remote.dvc.org/dataset-registry/a3/04af...
+```
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+## Description
+
+Returns the URL string of the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target file or directory
+(<abbr>artifact</abbr>), specified by its `path` in a `repo` (<abbr>DVC
+project</abbr>), is stored. The URL is formed by reading the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below) as
+well as the project's
+[remote configuration](/doc/command-reference/config#remote).
+
+The current project is used by default (the current working directory tree is
+walked up to find it), unless a `repo` argument is supplied.
+
+Unless a `remote` argument is given, the
+[default remote](/doc/command-reference/remote/default) of `repo` is used. The
+URL schema returned depends on the
+[type](/doc/command-reference/remote/add#supported-storage-types) of the remote
+used.
+
+⚠️ This function does not check for the actual existence of the target data in
+the remote storage.
+
+For Git-tracked <abbr>DVC repositories</abbr>, unless a `rev` argument is
+provided, the repo's `HEAD` version is used.
+
+💡 Having the resource's URL, it should be possible to download it directly with
+an appropriate library, such as
+[`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Object.download_fileobj)
+or
+[`paramiko`](https://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.get).
```

Comment:
> it can be downloaded with simple tools like curl or wget as well, not even libraries only
From https://github.com/iterative/dvc.org/pull/908#discussion_r384897583

I realize, but why would you use `get_url()` in a Python script to print this value and then use a CLI tool to get it? You can just use `dvc get --show-url` for that 🙂

Reply:
good point, there are cases though: E.g. you send it via rest API to some front-end and it users can use it directly to do whatever they want. But you are right, we can keep it this way.